### PR TITLE
1666674 for 4x

### PR DESCRIPTION
**Pull request by jcantrill**
fix https://bugzilla.redhat.com/show_bug.cgi?id=1666674 for 4.x

### CODE REVIEW

**Comment by richm** [comment]
e2e-operator is not a test flake - looks like a real error in fluentd:
```
--- FAIL: TestClusterLogging (333.14s)
    --- FAIL: TestClusterLogging/clusterlogging-group (60.00s)
        --- FAIL: TestClusterLogging/clusterlogging-group/Cluster_with_fluentd (135.62s)
```

File: pkg/k8shandler/logging.go [hunk 1]
```diff
@@ -14,10 +15,15 @@ import (
 
 type ClusterLogging struct {
 	*logging.ClusterLogging
+	Runtime *runtime.OperatorRuntime
```
Comment (ewolinetz) [inline pkg/k8shandler/logging.go]: we're using a runtime?

Comment (jcantrill) [reply pkg/k8shandler/logging.go]: maybe bad naming choice but its just the interface to allow mocking the api calls

**Pull request merged**